utils/ort/src/test/kotlin/JavaBootstrapperTest.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (C) 2024 The ORT Project Authors (see <https://github.com/oss-review-toolkit/ort/blob/main/NOTICE>)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+package org.ossreviewtoolkit.utils.ort
+
+import io.kotest.core.spec.style.StringSpec
+import io.kotest.matchers.shouldBe
+
+class JavaBootstrapperTest : StringSpec({
+    "The current JDK should be detected as a JDK" {
```

Comment:
The test name is confusing, I don't understand if this is a test for `isRunningOnJdk` or for `Environment.JAVA_VERSION`.

Reply:
The point of this test is that a JDK should be detected as a JDK, and not as a JRE. As tests are always run with a JDK, the test should succeed.

Reply:
Ok, that clarifies it for me. Maybe mentioning the function name in the test name would have helped, but that's not so important.